traffic_control/models.py

```diff
@@ -595,3 +595,146 @@ class Meta:
 
     def __str__(self):
         return "%s %s %s" % (self.id, self.code, self.value)
+
+
+class Barrier(Enum):
```

Comment:
Should this be like `BarrierType` or similar, because it might be confusing otherwise since we have `BarrierPlan` and `BrarrierReal` models also.